project/Dependencies.scala

```diff
@@ -1,120 +1,117 @@
 import sbt._
 
 object Dependencies {
-  private val akkaHttpCirceIntegrationV = "1.35.3"
-  private val akkaHttpV = "10.1.12" // scala-steward:off (CROM-6619)
-  private val akkaV = "2.5.31" // scala-steward:off (CROM-6637)
+  private val akkaHttpCirceIntegrationV = "1.37.0"
+  private val akkaHttpV = "10.1.14" // scala-steward:off (CROM-6619)
+  private val akkaV = "2.5.32" // scala-steward:off (CROM-6637)
   private val aliyunBcsV = "6.2.4"
-  private val aliyunCoreV = "4.5.20"
-  private val aliyunCrV = "4.1.1"
-  private val aliyunOssV = "3.11.3"
-  private val ammoniteOpsV = "2.3.8"
-  private val apacheCommonNetV = "3.8.0"
+  private val aliyunCoreV = "4.5.25"
+  private val aliyunCrV = "4.1.2"
+  private val aliyunOssV = "3.13.1"
+  private val ammoniteOpsV = "2.4.0"
   private val apacheHttpClientV = "4.5.13"
-  private val awsSdkV = "2.15.41" // scala-steward:off (CROM-6776)
+  private val awsSdkV = "2.17.29"
```

Comment:
Will need to make sure mutect2 test passes post-merge.

Reply:
kicked off AWS build 104 to get Jenkins' opinion before merge

Reply:
> kicked off AWS build 104 to get Jenkins' opinion before merge

FYI [104](https://fc-jenkins.dsp-techops.broadinstitute.org/job/cromwell-cron-aws/104/console) died due to the agent running out of space: `java.io.IOException: No space left on device`

Long live [106](https://fc-jenkins.dsp-techops.broadinstitute.org/job/cromwell-cron-aws/106/)

Reply:
Update: [106](https://fc-jenkins.dsp-techops.broadinstitute.org/job/cromwell-cron-aws/106/) passed